rts/lra/lra-test/lra-test-arquillian-extension/src/main/java/io/narayana/lra/arquillian/ConfigAuxiliaryArchiveAppender.java

```diff
@@ -38,25 +38,28 @@
 
     @Override
     public Archive<?> createAuxiliaryArchive() {
-        JavaArchive archive = ShrinkWrap.create(JavaArchive.class)
+        JavaArchive archive = ShrinkWrap.create(JavaArchive.class);
+
         // adding LRA spec interfaces under the Thorntail deployment
-                .addPackages(true, org.eclipse.microprofile.lra.annotation.Compensate.class.getPackage());
-        // adding Narayana LRA implementation under the WildFly Swarm deployment
+        archive
+            .addPackages(true, org.eclipse.microprofile.lra.annotation.Compensate.class.getPackage());
+
+        // adding Narayana LRA implementation under the Thorntail deployment
         archive.addPackages(true, io.narayana.lra.client.NarayanaLRAClient.class.getPackage())
-                .addPackages(true, io.narayana.lra.Current.class.getPackage())
-                .addPackage(LRACDIExtension.class.getPackage())
-                .addAsResource("META-INF/services/javax.enterprise.inject.spi.Extension")
-                .addClass(org.jboss.weld.exceptions.DefinitionException.class)
-                .addAsManifestResource(new StringAsset("<beans version=\"1.1\" bean-discovery-mode=\"annotated\"></beans>"), "beans.xml");
+            .addPackages(true, io.narayana.lra.Current.class.getPackage())
```

Comment:
I think this is code formatting, if it's not required I would favour not adding as it would aid readability